### PR TITLE
Opinion Mashup

### PR DESCRIPTION
**Pull request by andre1050**
## What does this change?
Adds the Opinion Mashup variant to the codebase as Variant B. Previous variants B and C have been kept but renamed to variants Y and Z.

## Why?
Because we want to iterate on the Opinion email and test a design that includes our best components yet.

## Link to supporting Trello card
https://trello.com/c/fCl9tw05/94-dev-test-new-best-of-opinion

### CODE REVIEW

File: src/components/cards/CommentCard.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+import { Headline } from "../../components/Headline";
+import { Image } from "../../components/Image";
+import { headline } from "../../styles/typography";
+
+type Size = "small" | "large";
+
+const tdStyle: TdCSS = {
+    backgroundColor: palette.neutral[100],
+    borderTop: `1px solid ${palette.opinion.main}`,
+    padding: "0"
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: `padding: "0"` unnecessary - it's added as default as we're using the TableRow component.

File: src/components/cards/CommentCard.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    width: "30%",
+    verticalAlign: "bottom",
+    padding: "0"
+};
+
+interface Props {
+    content: Content;
+    salt: string;
+    size: "large" | "small";
+    shouldShowImage?: boolean;
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: This prop can be deleted

File: src/components/cards/CommentCard.tsx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+const TrailText: React.FC<{
+    text: string;
+    linkURL: string;
+    size: Size;
+}> = ({ text, linkURL }) => {
+    return (
+        <td className="m-pad" style={standfirstStyle}>
+            <a style={linkStyle} href={linkURL}>
+                {" "}
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: Review whether this space is needed - don't think it is

File: src/components/cards/CommentCard.tsx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+                <span style={spanStyle}>{text}</span>
+            </a>
+        </td>
+    );
+};
+
+// TODO add alt text
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: Don't think this comment is needed - looks like the image already has a alt text

File: src/components/cards/CommentCard.tsx [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+        return null;
+    }
+
+    const formattedImage = formatImage(src, salt, width);
+    return (
+        <Image
+            src={formattedImage}
+            width={width}
+            alt={alt}
+            pillar="Opinion"
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: I don't think there's a good reason for this card to have anything hardcoded to the Opinion pillar. We have access to the pillar here, so we could pass it around as prop so we ensure any borders will render the appropriate pillar colour. This makes the card much more reusable - even if we don't have a use case for it outside of Opinion yet.

File: src/components/cards/CommentCard.tsx [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+                salt={salt}
+                src={contributorImageSrc}
+                alt={contributorImageAlt}
+            />
+        </td>
+    );
+
+    if (trailText && contributorImageSrc) {
+        return (
+            <RowCell tdStyle={{ padding: "0" }}>
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: Padding can be deleted

File: src/components/cards/CommentCard.tsx [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+        );
+    } else if (trailText) {
+        return (
+            <tr>
+                <TrailText text={trailText} linkURL={linkURL} size={size} />
+            </tr>
+        );
+    } else if (contributorImageSrc) {
+        return (
+            <RowCell tdStyle={{ padding: "0" }}>
```
Comment (andre1050) [inline src/components/cards/CommentCard.tsx]: Padding can be deleted

File: src/components/cards/CommentCardB.tsx [hunk 8]
```diff
@@ -97,7 +97,7 @@ const SupplementaryMeta: React.FC<{
     trailText: string;
     linkURL: string;
     contributorImageSrc?: string;
-    contributirImageAlt?: string;
+    contributorImageAlt?: string;
```
Comment (andre1050) [inline src/components/cards/CommentCardB.tsx]: Nice!

File: src/components/cards/HeadlineCard.tsx [hunk 9]
```diff
@@ -49,27 +50,41 @@ interface Props {
...
 const brazeParameter = "?##braze_utm##";
 
+const imgStyles: ImageCSS = {
+    verticalAlign: "middle",
+    border: "0"
+};
+
+const arrowColStyles = {
+    width: "23px",
+    paddingLeft: "10px",
```
Comment (andre1050) [inline src/components/cards/HeadlineCard.tsx]: Worth refactoring to `padding: 0 10px;`

File: src/fronts/opinion/variantB/Collections.tsx [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
 import { Collection as ICollection } from "../../../api";
 import { TableRowCell } from "../../../layout/Table";
-import { CommentCollection } from "./components/CommentCollection";
-import { EditorialCollection } from "./components/EditorialCollection";
-import { MediaCollection } from "./components/MediaCollection";
+import { GenericCollection } from "../../../collections/GenericCollection";
+import { InstagramCollection } from "../../../collections/InstagramCollection";
 import { LinkCollection } from "./components/LinkCollection";
-import { DefaultCollection } from "./components/DefaultCollection";
+// import { LinkCollection } from "../variantY/components/LinkCollection";
```
Comment (andre1050) [inline src/fronts/opinion/variantB/Collections.tsx]: Delete commented line

File: src/components/cards/CommentCard.tsx (hunk 1 above)
Comment (Krystian-gnm) [reply src/components/cards/CommentCard.tsx]: padding removed

**Pull request merged**